src/services/appwrite/database.ts

```
@@ -0,0 +1,62 @@
+		databaseId: string,
+		collectionId: string,
+		documentId: string,
+		data?: object,
+		permissions?: string[]
+	): Promise<Document> {
+		const result = await super.updateDocument<Document>(databaseId, collectionId, documentId, data, permissions)
+		await trackDatabaseOperation('update', collectionId, documentId)
+		return result
+	}
+
+	async deleteDocument(databaseId: string, collectionId: string, documentId: string): Promise<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)